src/machine/CoffeeMachine.java

```diff
@@ -0,0 +1,111 @@
+package machine;
+
+import java.util.Scanner;
+
+class CoffeeMachine {
+
+    private int amountOfWater;
+    private int amountOfMilk;
+    private int amountOfCoffeeBeans;
+    private int amountOfDisposableCups;
+    private int money;
+    private CoffeeMachineState currentState;
+
+    CoffeeMachineState getCurrentState() {
+        return this.currentState;
+    }
+
+    CoffeeMachine(int amountOfWater, int amountOfMilk, int amountOfCoffeeBeans,
+                         int amountOfDisposableCups, int money, CoffeeMachineState currentState) {
```

Comment:
1. It is better to set initial `currentState` right inside the constructor. Ideally this state should be changed/set only from inside the `CoffeeMachine`
2. minor: fix the alignment